interface/src/common/statusBar/statusBar.js

```diff
@@ -0,0 +1,12 @@
+angular.module('statusBar', [])
+
+.directive('statusBar', function() {
+  return {
+    restrict: 'E',
+    scope: {
+      msgText: '='
```

Comment:
I just tried converting from `msgText: '='` to use `'@'` instead. Whether or not I also make the corresponding change you mention to the HTML, the changes break the whole `statusBar`. I am not sure why yet, but the message text does not appear (the uninterpreted variable name appears instead) and the status bar does not disappear when it should either.

I don't have time to debug this now, so I'm reverting to using `'='` since that works and I'm making a note to follow up in the issue I just created.